app/views/comments/new.html.erb

```diff
@@ -1,11 +1,8 @@
-<div>
-    <h2> Create Comment </h2>
-    <%= form_with model: @comment, url: user_post_comments_path, data: { turbo: false} do |form| %>
-        
-        <div class="mb-3">
-            <%= form.label :text, class: "form-label" %>
-            <%= form.text_field :text, class: "form-control", id: "exampleFormControlInput1", placeholder: "Post Comment" %>
-        </div>
-        <%= form.submit "Post Comments", class:"btn btn-primary" %>
-    <% end %>
-</div>
+<%= form_with model: @comment, url: user_post_comments_path do |form| %>

```

Comment:
**You did great job!**

- [ ] Your app displays another error after previous reviewers request. If I click `See more` button in users post page. It shows me this error. Kindly fix this error and resubit again :)

<img width="788" alt="Screenshot 2023-05-14 at 00 11 14" src="https://github.com/techEmprez/Blog_app/assets/69011963/b4e5dcdc-e9d1-40f2-b4d2-bc6d2a170991">

<img width="788" alt="Screenshot 2023-05-14 at 00 14 17" src="https://github.com/techEmprez/Blog_app/assets/69011963/31f0c4ad-3ea4-47c3-93f1-eb1aad8fb3ff">

Reply:
all done